bibformat/format_elements/bfe_INSPIRE_enhanced_marcxml.py

```diff
@@ -121,6 +122,14 @@ def get_matched_id(subfields):
     return None
 
 
+def salvage_deleted_record_from_history(recid):
+    return create_record(decompress(run_sql("SELECT marcxml FROM hstRECORD WHERE id_bibrec=%s ORDER BY job_date DESC LIMIT 1", (recid, ))[0][0]))[0]
```

Comment:
Indeed hstRECORD is as good as bibfmt (for non deleted records), because bibupload writes in both tables at the same time (of course since it's MyISAM, non transactional it could crash in the middle, but meh...)